presto-main/src/main/java/com/facebook/presto/server/TaskResource.java

```diff
@@ -324,6 +324,17 @@ public void abortResults(@PathParam("taskId") TaskId taskId, @PathParam("bufferI
         taskManager.abortTaskResults(taskId, bufferId);
     }
 
+    @DELETE
+    @Path("{taskId}/remoteSource/{remoteSourceTaskId}")
```

Comment:
nit: Usually, in REST, the dash separated URLs are prefered. e.g.: `remote-source`. It doesn't matter much, just make sure you use the same one.

Reply:
@arhimondr : Interesting.. previously @nezihyigitbasi  suggests to use camel form.. https://github.com/prestodb/presto/pull/12488#discussion_r266687845

Reply:
I would rename back to `remote-source` for now. I also feel dash separated seems more often in URL. Is there any document about dash separator ? :)

Reply:
I don't think there's any strict guidelines of what to use. 

I read some interesting article about URLs before, where it was suggested to use hyphen as a separator. Unfortunately i cannot find a link to it.

The argument against camel case was that the first part of the URL (protocol and domain) is case insensitive, and although in the standard the remaining part of the URL is case sensitive - it is still recommended to keep it always lower case.  

The argument against underscore was that it is less readable for a non programmer and that it is harder to enter (SHIFT + '_' vs '-').

This mostly applies to URL's that are supposed to be entered by human. Most likely the `remote-source` API won't ever be used directly by a human. But some of the URL's from the REST API in Presto are very often used by humans. So, i would recommend to keep the namings consistent.